docs/oid-info.md

```diff
@@ -118,7 +118,7 @@ is formatted to the RFC 5280 specification as a DER-encoded string.
 
 ### 1.3.6.1.4.1.57264.1.9 | Build Signer URI
 
-Reference to specific build instructions that are responsible for signing. SHOULD be fully qualified. MAY be the same as Build Config URI. Build Signer URI is also included in the Subject Alternative Name.
```

Comment:
What do you think about including a line somewhere (maybe under the MUST section) on suggestions for what the SAN should be, e.g. "The Subject Alternative Name should be a stable identifier", something like that?

Reply:
Yeah I was thinking about this. This probably deserves its own section with good/bad examples. I'll tackle this in another PR, since I suspect this might be a bit lengthy.

Reply:
SG, thanks!